src/utils.ts

```diff
@@ -361,16 +361,37 @@ function getCacheMode(flags: Flags): "offline" | boolean {
         "online"
       )}   Use a locally cached module if available and fresh.\n\t${c.bold(
         "offline"
-      )}  Use a locally cached module if available, even if stale.\n\t${c.bold(
+      )}   Use a locally cached module if available, even if stale.\n\t${c.bold(
         "no-cache"
-      )} Never use the local cache.`
+      )}   Never use the local cache.`
     );
 
   if (flags.cache === "offline") return "offline";
   if (flags.cache === "online") return true;
   return false;
 }
 
+const validPreloadModes = ["static", "dynamic", "no-preloads"];
+function getPreloadMode(flags: Flags): boolean | string {
+  if (!flags.preload) return "static";
```

Comment:
When no flag is set, we shouldn't do any preloading at all.

Reply:
Mmm, think I misunderstood what you meant by defaulting to static. You mean default to static only if the flag is present at all?

Reply:
Yes exactly, preloading shouldn't be blanket enabled by default.